include/itkImageToTorchImageFilter.hxx

```diff
@@ -18,48 +18,76 @@
 #ifndef itkImageToTorchImageFilter_hxx
 #define itkImageToTorchImageFilter_hxx
 
-#include "itkImageToTorchImageFilter.h"
+#include <torch/torch.h>
 
-#include "itkImageRegionIterator.h"
-#include "itkImageRegionConstIterator.h"
+#include "itkImageToTorchImageFilter.h"
 
 namespace itk
 {
 
-template <typename TInputImage, typename TOutputImage>
-ImageToTorchImageFilter<TInputImage, TOutputImage>
-::ImageToTorchImageFilter()
-{}
+template< typename TInputImage, typename TOutputTorchImage >
+void
+ImageToTorchImageFilter< TInputImage, TOutputTorchImage >
+::PrintSelf( std::ostream &os, Indent indent ) const
+{
+  Superclass::PrintSelf( os, indent );
+}
+
+
+template< typename TInputImage, typename TOutputTorchImage >
+void
+ImageToTorchImageFilter< TInputImage, TOutputTorchImage >
+::GenerateInputRequestedRegion()
```

Comment:
You probably don't need to override this method.

Reply:
I will move the logic from `GenerateInputRequestedRegion` to `VerifyPreconditions`.  I will no longer override the former.